### DISCUSSION TITLE
ADStressDivergenceRZTensors

### DISCUSSION THREAD

**Discussion by hamedattariani** [idaholab/moose#26660] [Q&A Modules: Solid mechanics]
Hi,

I am trying to model an axisymmetric problem in cylindrical coordinates via ADStressDivergenceRZTensors.
So, it seems that the x direction represents the radial direction and y is z direction of a cylinder. How can I access to hoop stress, sigma_tt, in this case?

**Comment by GiudGiud** [comment]
Hello

You should be able to pass `generate_output = 'hoop_stress'` to the tensor mechanics master action to generate this output

Guillaume
